hyperliquid/utils/types.py

```diff
@@ -29,6 +29,14 @@
 )
 SpotMeta = TypedDict("SpotMeta", {"universe": List[SpotAssetInfo], "tokens": List[SpotTokenInfo]})
 
+
+SpotdAssetCtxsTokenInfo = TypedDict("SpotdAssetCtxsTokenInfo", {"name": str, "szDecimals": int, "weiDecimals": int})
+SpotdAssetCtxsUniverseInfo = TypedDict("SpotdAssetCtxsUniverseInfo", {"name": str, "tokens": List[int]})
+SpotdAssetCtxsMarketDataInfo = TypedDict("SpotdAssetCtxsMarketDataInfo", {"dayNtlVlm": str, "markPx": str, "midPx": str, "prevDayPx": str})
+
+SpotMetaAndAssetCtxsDict = TypedDict("SpotMetaAndAssetCtxsDict", {"tokens": List[SpotdAssetCtxsTokenInfo], "universe": List[SpotdAssetCtxsUniverseInfo]})
```

Comment:
no need to define this, this is just SpotMeta from above

Reply:
Did you try sending the requests and looking at the responses? We probably missed updating some documentation

Reply:
I tried just now and you are write. Updated info.py to reflect what comes as a response.